src/screens/Settings/About/PrivacyPolicyRow.js

```diff
@@ -0,0 +1,42 @@
+/* @flow */
+import React, { PureComponent } from "react";
+import { connect } from "react-redux";
+import { compose } from "redux";
+import { translate } from "react-i18next";
+import { View, Linking } from "react-native";
+import { createStructuredSelector } from "reselect";
+import SettingsRow from "../../../components/SettingsRow";
+import type { T } from "../../../types/common";
+import colors from "../../../colors";
+import { urls } from "../../../config/urls";
+import ExternalLink from "../../../images/icons/ExternalLink";
+
+const mapStateToProps = createStructuredSelector({});
```

Comment:
we probably don't need it

Reply:
yea, you are right. I leave it for some that would probably need it, but the ones that are just for redirection/simple it is not needed